### DISCUSSION TITLE
Addon Version 8.2 available for Chrome and Edge

### DISCUSSION THREAD

**Discussion by paulrouget** [aclap-dev/video-downloadhelper#68] [Announcements]
Changelog available here: https://www.downloadhelper.net/changelog/google/stable

Firefox version is still being reviewed by Mozilla.

**Comment by Maister-G** [comment]
DownloadHelper 8.2.0.19 (stable) en Chrome, al intentar descargar algun video aparece el siguiente Error: No download strategy for builder: undefined (detalles) at yT (chrome-extension://lmjnegcaeklhafolokijcfjliaokphfk/background/main.js:44:6803)
    at async Zr (chrome-extension://lmjnegcaeklhafolokijcfjliaokphfk/background/main.js:44:12666)

**Comment by GrampaWildWilly** [comment]
In the middle of your comment, I see this:

yT

Are you by any chance trying to download something from YouTube?  You do know, I hope, that Google does not permit non-Google extensions in the Chrome web store that download from YouTube.  It's against their rules.  If you want to download from YouTube using VDH, you can't do it with VDH for Chrome.  VDH for Firefox & VDH for Edge can download from YouTube, but not VDH for Chrome.

But if that's not your issue, then just ignore me & wait for Paul to come in here & explain things.  You see, I don't work for VDH, but Paul is the one writing the code.

**Reply by Maister-G** [reply]
Concretamente me ha ocurrido en los servidores Gamovideo y Powvideo.

**Reply by GrampaWildWilly** [reply]
You should open a new discussion by following these instructions.  Be sure to follow all the steps:

[How to post a proper problem report](https://github.com/aclap-dev/video-downloadhelper/wiki/I-need-help)

**Reply by paulrouget** [reply]
That's a bug. @Maister-G re-reported here: https://github.com/aclap-dev/video-downloadhelper/discussions/71
